config/pipelines/high_throughput_scrna_core_cdna_prep.wip.yml

```diff
@@ -26,3 +26,24 @@ scRNA Core cDNA Prep Input:
     LRC HT 5p Chip: LRC HT 5p GEMs
     LRC HT 5p GEMs: LRC HT 5p cDNA PCR
     LRC HT 5p cDNA PCR: LRC HT 5p cDNA PCR XP
+scRNA Core Chromium HT Aggregation:
```

Comment:
When we came up with the naming before, we didn't include 'scRNA Core'.

See all names planned out here: https://github.com/sanger/limber/issues/1444  

Don't feel strongly about it though. We did include '5p' though, which seems like it could matter.

Reply:
I added scRNA Core because it's going to be aggregating with scRNA Core plate purposes. It's not generic to just any aggregation. I feel this is correct in terms of Limber naming conventions.

The **5p** I was less sure about, and initially had it in there.
I think I will re-add it. Say we add a 3p chip later, this means we will need a separate aggregation pipeline for the 3p plate purposes.

Reply:
Added 5p